_slides/07-Ping-LAC.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption here —Ping
```

Comment:
Hi Ping... nice to see you in this training... 
I'm reviewing your code.  It looks great!  Lol!